src/coreclr/src/jit/lclvars.cpp

```diff
@@ -2670,7 +2670,49 @@ void Compiler::lvaSetStruct(unsigned varNum, CORINFO_CLASS_HANDLE typeHnd, bool
         compGSReorderStackLayout = true;
         varDsc->lvIsUnsafeBuffer = true;
     }
+#ifdef DEBUG
+    if (doExtraSuperPmiQueries)
```

Comment:
Do you need the `doExtraSuperPmiQueries` variable at all? Couldn't you just call `JitConfig.EnableExtraSuperPmiQueries()` here?